charts/datadog/templates/_container-otel-agent.yaml

```diff
@@ -75,6 +75,12 @@
     - name: DD_OTELCOLLECTOR_ENABLED
       value: "true"
     {{- end }}
+    {{- if .Values.otelAgentGateway.enabled }}
+    - name: DD_OTELCOLLECTOR_CONVERTER_FEATURES
+      # Exclude infra attribute or prometheus in Daemonset in Gateway setup.
+      # Users must explicitly add infra attribute and prometheus if they need in Gateway, to avoid potential duplications
+      value: "health_check,zpages,pprof,ddflare"
```

Comment:
It should be `datadog`: https://github.com/DataDog/datadog-agent/pull/40921

Reply:
sorry, i meant this for here: https://github.com/DataDog/helm-charts/pull/2018/files#diff-14d01852cea616f9bfcddc7d61f5819b071a0933ce469bec50476794d2c2eaefR174